lib/hexpm/accounts/user.ex

```diff
@@ -41,8 +42,13 @@ defmodule Hexpm.Accounts.User do
     |> update_change(:password, &Auth.gen_password/1)
   end
 
-  def build(params, confirmed? \\ not Application.get_env(:hexpm, :user_confirm)) do
-    changeset(%Hexpm.Accounts.User{}, :create, params, confirmed?)
+  def build_organization(organization) do
+    change(%User{username: organization.name}, %{})
```

Comment:
I tested locally and we were not filling in the FK, this seems to fix it:

```suggestion
    change(%User{username: organization.name, organization_id: organization.id}, %{})
```